streaming/base/util.py

```diff
@@ -222,7 +225,8 @@ def retry(exc_class: TCallable) -> TCallable:
     ...
 
 
-# error: Type "(TCallable@retry) -> TCallable@retry" cannot be assigned to type "(func: Never) -> Never"
+# error: Type "(TCallable@retry) -> TCallable@retry" cannot be assigned to type
+# "(func: Never) -> Never"
 def retry(  # type: ignore
```

Comment:
Do you want to use this package instead? https://pypi.org/project/retry/

Reply:
It is a stale project where last update was in 2016. Hence, it would be risky to use it.